docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,154 @@
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [development build](https://docs.expo.dev/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase. A development build supports the same workflows as Expo Go, allowing you to develop from your local machine or test updates via a QR code, except it is compatible with all of the native code you include in your standalone app.
+
+[Follow these instructions for creating a development build](https://docs.expo.dev/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
+```
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    }
+  }
+}
+```
+If you wish to run a development build on an iOS simulator instead of a device, your profile may also include:
+```
+{
+  "ios": {
+    "simulator": true
+  }
+}
+```
+`expo-dev-client` should be present in your **package.json**, as well. Once you run `eas build` with your development build profile and platfrom of choice, you will have an app binary file that you can install on your device or simulator.
+
+If you are already using a development build, just rebuild your app, as your native code dependencies have changed.
+
+## Test your changes
+
+Test your changes locally on your development build by running
+<Terminal cmd={[
+'npx expo start --dev-client'
+]} />
+Then follow the prompts on the terminal to run it on your simulator, or scan the QR code to run it on your device. Try an action that will register a Firebase Analytics event. Note that it may take a few minutes for events to show up in the Firebase Console.
+
+## Conclusion
+Now that you have your own development build, you can use any of the many React Native Firebase packages in your development and testing workflow. Not only that, but you're only minutes away from adding [any additional native functionality you'd like](https://docs.expo.dev/development/getting-started/#customizing-your-runtime).
+
+## Other migration scenarios
+
+### Analytics for Expo for web
+
+React Native Firebase only supports iOS and Android. If you previously used `expo-firebase-analytics` to send events from an Expo for web client, you will need to use the Firebase JavaScript SDK. In, the Firebase Console, go to Project Settings, and then your web app, and copy the code under "SDK setup and configuration." Then you can switch between React Native Firebase and Firebase JS requests using the `Platform` component:
```

Comment:
```suggestion
React Native Firebase only supports iOS and Android. If you previously used `expo-firebase-analytics` to send events from an Expo for web client, you will need to use the Firebase JavaScript (JS) SDK. In the Firebase Console, go to Project Settings, and then your web app, and copy the code under "SDK setup and configuration." You can switch between React Native Firebase and Firebase JS requests using the `Platform` component:

```

Should we add an instruction here on how to install Firebase JS SDK? Or may be we can link here as a note on how to do it and configure from "Using Firebase" guide that I am working on.